main.tf

```diff
@@ -0,0 +1,59 @@
+resource "vsphere_folder" "sdc_folder" {
+  path          = local.folder_path
+  type          = "vm"
+  datacenter_id = data.vsphere_datacenter.datacenter.id
+}
+
+resource "vsphere_content_library" "sdc_content_library" {
+  name            = "${var.cdo_tenant_name}-content-library"
+  description     = "Content library to hold SDC OVAs for CDO tenant ${var.cdo_tenant_name}"
+  count           = var.sdc_content_library_item_id == null ? 1 : 0
```

Comment:
Lets put the count at the top of this resource like:

```hcl
resource "vsphere_content_library" "sdc_content_library" {
  count = var.sdc_content_library_item_id == null ? 1 : 0

  name            = "${var.cdo_tenant_name}-content-library"
  description     = "Content library to hold SDC OVAs for CDO tenant ${var.cdo_tenant_name}"
  storage_backing = [data.vsphere_datastore.datastore.id]
}
```

Reply:
Done.